CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Update roleSlug
```

Comment:
Can you be clearer here? The idea is to explain the change in a few words, so it could be something like:

```suggestion
- Fixed an error happening when fetching a user's role slug.
```